### PR TITLE
Extract nuget push from build pipeline

### PR DESCRIPTION
**Pull request by paveliam**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -137,28 +137,11 @@ steps:
...
     TargetFolder: '$(Build.ArtifactStagingDirectory)'
   condition: succeededOrFailed()
-
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish copied artifacts'
-  inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
-    ArtifactName: 'drop'
-    publishLocation: 'Container'
-  condition: succeededOrFailed()
```
Comment (mialeska) [inline azure-pipelines.yml:147]: oh no, these artifacts(log and screens) should be published unconditionally, not just for master source branch

**Review by mialeska — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (paveliam) [reply azure-pipelines.yml:147]: Why do you need them? How are you going to use these artifacts?

Comment (mialeska) [reply azure-pipelines.yml:147]: logs can help to detect failure reasons, especially when previous test did smth which caused the failure of the next test

Comment (paveliam) [reply azure-pipelines.yml:147]: changes reverted

**Review by mialeska — APPROVED** [review]

**Pull request merged**